tests/TWRippleTests.cpp

```diff
@@ -0,0 +1,48 @@
+// Copyright © 2017-2019 Trust.
+//
+// This file is part of Trust. The full Trust copyright notice, including
+// terms governing use, modification, and redistribution, is contained in the
+// file LICENSE at the root of the source code distribution tree.
+
+#include "gtest/gtest.h"
+#include "TWTestUtilities.h"
+
+#include <TrustWalletCore/TWRippleAddress.h>
+#include <TrustWalletCore/TWHash.h>
+#include <TrustWalletCore/TWHDWallet.h>
+#include <TrustWalletCore/TWPrivateKey.h>
+
+TEST(Ripple, ExtendedKeys) {
+    auto wallet = WRAP(TWHDWallet, TWHDWalletCreateWithMnemonic(
+        STRING("ripple scissors kick mammal hire column oak again sun offer wealth tomorrow wagon turn fatal").get(),
```

Comment:
is there a way to reuse this test phrase from other files? we already have this somewhere else